tfx/dsl/component/experimental/function_parser.py

```diff
@@ -21,9 +21,9 @@
 
 import enum
 import inspect
+import sys
```

Comment:
I think this is an unused import.

Reply:
addressed [here](https://github.com/tensorflow/tfx/pull/5498/commits/303bb6f10748cbc4f46619fd5c052965e86623c8)